java/code/src/com/suse/manager/utils/PagedSqlQueryBuilder.java

```diff
@@ -306,7 +307,8 @@ public <T extends BaseTupleDto> DataResult<T> run(Map<String, Object> parameters
         String sortSql = "";
         if (pageControl.map(PageControl::getSortColumn).orElse(null) != null) {
             String sortDirection = pc.isSortDescending() ? "DESC" : "ASC";
-            sortSql = String.format(" ORDER BY %s %s", pc.getSortColumn(), sortDirection);
+            String sortColumn = sanitizeColumnNames(pc.getSortColumn());
+            sortSql = String.format(" ORDER BY %s %s", sortColumn, sortDirection);
```

Comment:
instead of passing it as a string and use java string format, why not use Hibernate query parameters, like we do for all the other fields, and add the sort column to the `parameters` map, and let hibernate deal with the parameters replacement, which takes care of all the escapes? That would make the solution more reboust and would allow order by using any column name, no matter what name it is (or error in case the column doesn't exists)

Reply:
Hibernate sadly doesn't accept query parameters for fields, only values.